product/views.py

```diff
@@ -1,3 +1,94 @@
-from django.shortcuts import render
+from django.http                    import HttpResponse, JsonResponse
+from django.views                   import View
+from django.db.models               import Count, F
 
-# Create your views here.
+import json
+
+from .models          import Product, Category, SubCategory
+
+class ListView(View):
+    def get(self, request):
+        page = int(request.GET.get("page",1))
+        category_code = request.GET.get("cateCd")
```

Comment:
```suggestion
        category_code = request.GET.get("category_code")
```
참고하는 대상에서 그런 코드명을 썼다고 나도 그렇게 해야할 필요는 없습니다. 정확한 변수명 사용을 해주세요.